ivy/functional/backends/tensorflow/activations.py

```diff
@@ -46,7 +46,8 @@ def relu(x: Tensor, /, *, complex_mode="jax", out: Optional[Tensor] = None) -> T
     return tf.nn.relu(x)
 
 
-def sigmoid(x: Tensor, /, *, out: Optional[Tensor] = None) -> Tensor:
+@with_unsupported_dtypes({"2.13.0 and below": ("complex",)}, backend_version)
```

Comment:
Why are we marking complex as unsupported for this?

Reply:
I think we has this discussion earlier. Even though `tensorflow` does support complex dtypes, We don't get the same results when compared with `jax`. A quick fix for this issue was the `sigmoid_jax_like` function.

The implementation of the sigmoid function in the `tensorflow` backend is correct, but does not match the result from `jax`

Reply:
That's a problem with tensorflow (this issue here https://github.com/tensorflow/tensorflow/issues/61800) which will be fixed once they solve it on their end. I think marking it as an unsupported dtype is the wrong move, it might be better to leave it as supported but change the ground truth backend in the test to, for example, jax so that only the tf backend test will fail